core/deployment/src/main/java/io/quarkus/deployment/configuration/ConfigDefinition.java

```diff
@@ -71,6 +71,10 @@
             .asList(QUARKUS_NAMESPACE + ".live-reload.password", QUARKUS_NAMESPACE + ".live-reload.url",
                     QUARKUS_NAMESPACE + ".debug.generated-classes-dir", QUARKUS_NAMESPACE + ".debug.reflection",
                     QUARKUS_NAMESPACE + ".build.skip",
+                    QUARKUS_NAMESPACE + ".platform.group-id",
+                    QUARKUS_NAMESPACE + ".platform.artifact-id",
+                    QUARKUS_NAMESPACE + ".platform.version",
+                    QUARKUS_NAMESPACE + "-plugin.version",
```

Comment:
I'm pretty sure that this entry isn't needed.  The config subsystem should only warn on properties whose name starts with `quarkus.`

Reply:
Ok, removing it from here. Thanks.